validation/edepsim_validation.py

```diff
@@ -222,24 +230,24 @@ def MINERvA_bounds(i):
         plt.axes().set_aspect('equal')
         plt.hist2d(vertex['x_vert'], vertex['y_vert'], bins = 100)
```

Comment:
The (soon-to-be depricated) `vertex` dataset is unfortunately in units of mm still. Until we sort out the new genie datasets, lets at least put it in cm so it makes sense with the geometry markers: `vertex[coord]` --> `vertex[coord]/10.`

Reply:
Ah, missed that. Fixed.